src/presentation/controller/Products/ProductController.ts

```
@@ -155,21 +155,37 @@ export class ProductController {
+                }
+            }
+
             const request = {
                 ...req.body,
                 productId: req.params.id,
                 sizeOptions,
+                productOption,
                 image: req.file,
                 restaurantOwnerId: user.userId,
             };
             delete request.categoryId;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)